### ISSUE TITLE
Debugger doesn't show variables values (DOS/4GW)

### ISSUE THREAD

**Issue by viti95** [open-watcom/open-watcom-v2#692]
Hi, i'm trying to debug FastDoom (DOS/4GW) using this tutorial (https://www.javiergutierrezchamorro.com/depurar-aplicaciones-dos-con-watcom-c-y-openwatcom-c/), but all the variables in the debugger are shown empty. I'm currently using the latest build of OpenWatcom v2 (2021-06-04 01:45:02, Windows 64 bits) and DosBox-X (0.83.13 2021-04-30 23:16). The debugger is connected via serial port (com0com) and it's able to stop at breakpoints.

I've enabled all the debug features at compile time (/d2) and linker (all), so i don't know what's currently wrong. Some times the DOS/4GW game just crashes with a "Stack Overflow!" message only when running the debugger.

![2021-06-04 14_41_19-Greenshot](https://user-images.githubusercontent.com/8323882/120809337-3b6a3500-c54a-11eb-9894-98d51b0a7f59.png)

**Comment by jmalak** [comment]
Did you test it with OW 1.9 debugger?
OW2.0 debugging info is backward compatible with OW1.9 format.

**Comment by viti95** [comment]
No, all the testing I've done is with the OW2 debugger. But I'll try it and post the results. Thanks.

**Comment by viti95** [comment]
I was able to test with OW 1.9, but the problem remains exactly the same.

**Comment by jmalak** [comment]
Thanks for testing with OW 1.9.
It is strange and probably depend on application.
Are you able give me some compilable sample which exhibit this issue?

Anyway if you get "Stack Overflow!" from Application under Debugger you should setup stack size to bigger value.
If it is reported by Debugger then it will require change stack size for Debugger.

**Comment by jmalak** [comment]
I tried double stack size for OW DOS Debugger (from 12kB to 24 kB).
Please check with latest OW2 build.
